content/en/blog/_posts/2024-09-30-gateway-api-v1.2/index.md

```diff
@@ -0,0 +1,418 @@
+process -- but it also brings two breaking changes that you'll want to be
+careful of.
+
+## BREAKING CHANGES
+
+### GRPCRoute and ReferenceGrant `v1alpha2` Removal
+
+Now that the `v1` versions of GRPCRoute and ReferenceGrant have graduated to
+Standard, the old `v1alpha2` versions have been removed from both the Standard
+and Experimental channels, in order to ease the maintenance burden that
+perpetually supporting the old versions would place on the Gateway API
+community.
+
+Before upgrading to Gateway API v1.2, you'll want to confirm that any
+implementations of Gateway API have been upgraded to support the v1 API
+version of these resources instead of the v1alpha2 API version. Note that even
+if you've been using v1 in your YAML manifests, a controller may still be
+using v1alpha2 which would cause it to fail during this upgrade. Additionally,
+Kubernetes itself goes to some effort to stop you from removing a CRD version
+that it thinks you're using: check out [the release notes] for more
+information about what you need to do to safely upgrade.
+
+[the release notes]: https://github.com/kubernetes-sigs/gateway-api/releases/tag/v1.2.0
+
+### Experimental `Gateway.status.supportedFeatures`
+
+A much smaller breaking change, `Gateway.status.supportedFeatures` is now a
+list of objects instead of a list of strings. The objects have a single `name`
+field, so the translation from the strings is straightforward, but moving to
+objects permits a lot more flexibility for the future. This stanza is not yet
+present in the Standard channel.
+
+## Graduations to the Standard Channel
+
+Gateway API 1.2.0 graduates four features to the Standard channel. This means
+they are no longer experimental concepts; inclusion in the Standard release
+channel denotes a high level of confidence in the API surface and provides
+guarantees of backward compatibility. Of course, as with any other Kubernetes
+API, Standard Channel features can continue to evolve with backward-compatible
+additions over time, and we certainly expect further refinements and
+improvements to these new features in the future. For more information on how
+all of this works, refer to the [Gateway API Versioning
+Policy](https://gateway-api.sigs.k8s.io/concepts/versioning/).
+
+#### [HTTPRoute Timeouts](https://gateway-api.sigs.k8s.io/guides/http-routing/)
+
+[GEP-1742](https://gateway-api.sigs.k8s.io/geps/gep-1742/) introduced the
+`timeouts` stanza into HTTPRoute, permitting configuring basic timeouts for
+HTTP traffic. This is a simple but important feature for proper resilience
+when handling HTTP traffic, and it is now Standard.
+
+For example, this HTTPRoute configuration sets a timeout of 300ms for traffic
+to the `/face` path:
+
+```yaml
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: face-with-timeouts
+  namespace: faces
+spec:
+  parentRefs:
+    - name: my-gateway
+      kind: Gateway
+      port: 80
```

Comment:
Any reason to include port here? I think it may add unnecessary complexity to this example

Reply:
Removed.